mama/c_cpp/src/c/transport.c

```diff
@@ -215,6 +219,21 @@ init (transportImpl* transport, int createResponder)
           if (rval != MAMA_STATUS_OK) return rval;
     }
 
+    snprintf(searchName, sizeof(searchName), "mama.%s.dq.disabled", middleware);
+
+    dqDisabled = strtobool(mama_getProperty(searchName));
+
+    if(!dqDisabled)
+    {
+        snprintf(searchName, sizeof(searchName), "mama.%s.transport.%s.dq.disabled", middleware, self->mName);
```

Comment:
dq.disabled should be an #ifdef - actually change to dq_enabled 

And in fact change the variable to dqEnabled and flip the logic round - I personally hate double negatives in code, should have thought about this earlier